lib/session-store.js

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Let's get TypeScript checking us!

```suggestion
// @ts-check
/*
```